src/sentry/features/temporary.py

```diff
@@ -74,6 +74,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:autofix", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
     # Autofix use new strategy without codebase indexing
     manager.add("organizations:autofix-disable-codebase-indexing", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, default=False, api_expose=True)
+    # Enables Chonk UI
+    manager.add("organizations:chonk-ui", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
```

Comment:
is this the official name or should we just call it ui2?

Reply:
I dont know, I checked with Jesse and he said everyone pretty much references it by chonk. I also hate anything 2.0 naming related...

Reply:
I'm OK with chonk as an internal name, though we'll probably want something more branded for the actual rollout.